bin/check_metric_limits.rb

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ruby-2.1.1
```

Comment:
Alright sounds good.

Out of curiosity, does [this line in our gemspec](https://github.com/cisco/cisco-network-node-utils/blob/develop/cisco_node_utils.gemspec#L27) not force ruby 1.9.3 or higher on the ADS server?